inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcMultiBatchingOutputFormat.java

```diff
@@ -399,14 +420,44 @@ protected GenericRowData generateRecord(RowType rowType, Map<String, String> fie
                         record.setField(i, Double.valueOf(fieldValue));
                         break;
                     case TIME_WITHOUT_TIME_ZONE:
-                    case TIMESTAMP_WITHOUT_TIME_ZONE:
+                        // case TIMESTAMP_WITHOUT_TIME_ZONE:
                     case INTERVAL_DAY_TIME:
                         TimestampData timestampData = TimestampData.fromEpochMillis(Long.valueOf(fieldValue));
                         record.setField(i, timestampData);
                         break;
                     case BINARY:
                         record.setField(i, Arrays.toString(fieldValue.getBytes(StandardCharsets.UTF_8)));
                         break;
+
```

Comment:
Can L431-L460 also apply to CK、GP、Oracle、PG、SqlServer、TDSQL ？

Reply:
Didn't test for these databases.

Reply:
A little danger. While it doesn't matter